### PR TITLE
Adding documentation on stop parking

### PR DESCRIPTION
**Pull request by georgekronberg**
@twopointzero not sure who else should be on this PR
I fixed close to a 100 issues in Parking.apib - now it's green in terms of passing apib linter.
Quite a lot of changes here, some are not obvious, so don't hesitate to ask.

### CODE REVIEW

**Comment by georgekronberg** [comment]
btw, if we have apiary plus (or whatever their paid tier is called) we could view any branch on github via apiary, not just master. would be super useful for pull requests.

**Comment by georgekronberg** [comment]
@jdstuart hi JD, here're the docs for stop parking

File: Sessions.apib [hunk 1]
```diff
@@ -1,305 +1,305 @@
...
++ Response 400
+
+        {
+            "error": "The request is invalid"
+        }        
+
+## SessionStop [/public/parking/sessions/{parkingSessionId}/stop]
+An individual vehicle resource
+
+### Stop a parking session [PUT]
```
Comment (jdstuart) [inline Sessions.apib]: Should this not be a POST?

File: Sessions.apib [hunk 2]
```diff
@@ -1,305 +1,305 @@
...
+The *id* is assigned by the API at the moment of creation. 
+    
+**Possible Response Codes**
+
+- 204
+- 400
+
++ Request stop parking session
+
+        /public/parking/sessions/{parkingSessionId}/stop
```
Comment (jdstuart) [inline Sessions.apib]: This endpoint does not exits. - https://corp.paybyphone.com/jira/browse/SITH-1486

**Comment by georgekronberg** [comment]
@jdstuart sorry for the confusion, but the only file i changed was Parking.apib
All other changes you see are likely line-ending conversion
Github (surprisingly) doesn't give you to turn "ignore whitespace" option when viewing diff, unlike stash
But you can turn with feature on, by adding ?w=1 to the query string, like so https://github.com/paybyphone/apidocumentation/pull/20/files?w=1
One you open that link you'll see that Session.apib haven't changed and those endpoints you commented on were there. Even if that endpoint doesn't currently exist - fixing it is a separate ticket, unrelated to this one.
So please take a look at Parking.apib changes.

**Pull request merged**